pkgs/applications/misc/yaft/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub, pkgconfig, ncurses, readline }:
+
+stdenv.mkDerivation rec {
+  version = "0.2.9";
+  name = "yaft-${version}";
+
+  src = fetchFromGitHub {
+    owner = "uobikiemukot";
+    repo = "yaft";
+    rev = "v${version}";
+    sha256 = "0l1ig8wm545kpn4l7186rymny83jkahnjim290wsl7hsszfq1ckd";
+  };
+
+  installPhase = ''
+    mkdir -p $out/{bin,share/man}
+    DESTDIR=$out PREFIX=$out/bin/ MANPREFIX=$out/share/man/
+  '';
+
+  meta = {
+    homepage = "https://github.com/uobikiemukot/yaft";
+    description = "yet another framebuffer terminal";
```

Comment:
The "y" in "yet" should be [capitalized](https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes).